typescript/web/src/components/layout/top-bar/user-menu.tsx

```diff
@@ -180,8 +180,8 @@ const UserMenuGroup = () => {
       {status === "loading" && <SignOutMenuItem cursor="default" disabled />}
       {status === "authenticated" && (
         <>
-          <SignOutMenuItem />
-          <SettingsMenuItem />
+          <SettingsMenuItem /> 
+          <SignOutMenuItem /> 
```

Comment:
```suggestion
          <SettingsMenuItem />
          <SignOutMenuItem />
```

@Suveksha They are remaining spaces at the end of the lines. Which will make the style checks to fail. Can you please remove them?
To prevent such problems without having to take care of them, you can use the [VS Code](https://code.visualstudio.com/) editor and [install the recommended extensions](https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions) we've defined in this repository.

Reply:
I have removed the spaces. Please check it out.